java/ql/src/Security/CWE/CWE-200/TempDirLocalInformationDisclosureFromSystemProperty.ql

```diff
@@ -0,0 +1,82 @@
+/**
+ * @name Temporary Directory Local information disclosure
```

Comment:
This is clearly very similar to the other query. As far as I know, there is no requirement for queries to have unique `@names` within a language. It's helpful if there is some small disambiguation otherwise it looks like an error in contents lists of alerts or query help.

Reply:
Any suggestions on how to differentiate them?

Reply:
Clearly they both find the problem that you've used in the name - I don't really know enough about QL and programming to be able to make a good suggestion here. Possibly @aibaars might have some thoughts.